Riot/Modules/Room/RoomViewController.m

```diff
@@ -1152,10 +1151,29 @@ - (void)leaveRoomOnEvent:(MXEvent*)event
     [self notifyDelegateOnLeaveRoomIfNecessary];
 }
 
+
++ (Class) mainToolbarClass
+{
+    if (@available(iOS 15.0, *)) {
+        if (RiotSettings.shared.enableWysiwygComposer)
+        {
+            return WysiwygInputToolbarView.class;
+        }
+        else
+        {
+            return RoomInputToolbarView.class;
+        }
+    }
+    else
+    {
+        return RoomInputToolbarView.class;
+    }
+}
```

Comment:
I guess we removed iOS 15 dependency from the wysiwyg so should be save to remove this availability check?

Reply:
Ah yes you are correct, we probably just need to keep the one on the labs flag.